!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  filter_min_time: 1.0
  filter_timeout: 0.25
  filter_update_rate: 20
  filter_val_gate_thresh: 1.923
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      filter_min_time: 1.0
      filter_timeout: 0.25
      filter_update_rate: 20
      filter_val_gate_thresh: 1.923
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      kf_init_azim_stdev: 1.0
      kf_init_elev_stdev: 1.0
      kf_init_range_stdev: 1.0
      kf_init_speed_stdev: 2.0
      kf_q_azim_stdev: 20.0
      kf_q_elev_stdev: 20.0
      kf_q_speed_stdev: 5.0
      kf_r_azim_stdev: 20.0
      kf_r_elev_stdev: 20.0
      kf_r_range_stdev: 1.0
      kf_r_speed_stdev: 5.0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  kf_init_azim_stdev: 1.0
  kf_init_elev_stdev: 1.0
  kf_init_range_stdev: 1.0
  kf_init_speed_stdev: 2.0
  kf_q_azim_stdev: 20.0
  kf_q_elev_stdev: 20.0
  kf_q_speed_stdev: 5.0
  kf_r_azim_stdev: 20.0
  kf_r_elev_stdev: 20.0
  kf_r_range_stdev: 1.0
  kf_r_speed_stdev: 5.0
state: []
